changelogs/changelog.yaml

```diff
@@ -26,6 +26,7 @@ releases:
       - win_iis_website - Actually restart the site when ``state=restarted`` - https://github.com/ansible/ansible/issues/63828
       - win_partition - Fix invalid variable name causing a failure on checks - https://github.com/ansible/ansible/issues/62401
       - win_partition - don't resize partitions if size difference is < 1 MiB
+      - win_scoop - add checks for globally installed packages for better idempotency checks
```

Comment:
This needs to be in a separate file in `changelogs/fragments/<name>.yml`. The `changelog.yaml` file is auto generated when the next release is created based on the fragments that are present.